api/src/zammad.js

```diff
@@ -39,16 +40,29 @@ const api = async (path, options = {}) => {
 
 async function sync(doc, modelName) {
   try {
-    if (ENVIRONMENT !== "production") return;
+    //if (ENVIRONMENT !== "production") return;
```

Comment:
on ne sync pas la preprod ou le local
```suggestion
    if (ENVIRONMENT !== "production") return;
```